.github/workflows/release.yml

```diff
@@ -42,6 +42,7 @@ jobs:
 
       - name: "Install Java ${{ env.JAVA_VERSION }}"
         uses: actions/setup-java@v4
+        continue-on-error: true
```

Comment:
wait
why this?

Reply:
Thanks for asking.
For some reason, the release flow fails at the Post Install Task
For instance, here: https://github.com/hyperledger/identus-apollo/actions/runs/10279883067
The entire release fails even if the packages are published and the `chore` commit is pushed.

I could not find a solution for this now except to add `continue-on-error` to the `Post Install Java 17` task.

<img width="1060" alt="image" src="https://github.com/user-attachments/assets/740718b7-4858-48f1-a8d0-0518103b9e5c">

Reply:
https://github.com/hyperledger/identus-apollo/issues/189 for tracking